# start a planner

uint32 plannerid
---
Trajectory trajectory
